torchvision/transforms/_presets.py

```diff
@@ -53,11 +54,16 @@ def __init__(
         self.std = list(std)
         self.interpolation = interpolation
         self.antialias = antialias
+        self.backend = backend.lower()
+        if self.backend not in ("pil", "tensor"):
+            raise ValueError(f"backend parameter must be pil or tensor, got {backend}")
 
     def forward(self, img: Tensor) -> Tensor:
+        if self.backend == "tensor":
+            img = F.pil_to_tensor(img)
         img = F.resize(img, self.resize_size, interpolation=self.interpolation, antialias=self.antialias)
         img = F.center_crop(img, self.crop_size)
-        if not isinstance(img, Tensor):
+        if self.backend == "pil":
```

Comment:
This is not equivalent. Previously, we allowed users to pass tensors and PIL images without setting anything else. This is why the smoke tests fails right now https://github.com/pytorch/vision/actions/runs/5079158848/jobs/9124559609?pr=7629#step:13:13110:

We pass in a tensor

https://github.com/pytorch/vision/blob/4125d3a02b15faf4b19767a91797320151ce8bc6/test/smoke_test.py#L54

but are not setting `backend="pil"`:

https://github.com/pytorch/vision/blob/4125d3a02b15faf4b19767a91797320151ce8bc6/test/smoke_test.py#L61-L65

Reply:
OK, I'll remove changes to those preset and handle that in the references instead